tests/test_auth.py

```diff
@@ -3,13 +3,12 @@
 description: Tests signature calculation and verification
 author: Oliver Zscheyge <oliverzscheyge@gmail.com>
 '''
-
 from tests.context import epages
 
-CODE = 'rp6YlWEcSxS81OTmdksLOaqv7TRbpzSV'
-ACCESS_TOKEN_URL = 'https://devshop.epages.com/rs/shops/lingering-hill-2037/token'
-CLIENT_SECRET = 'pM8vwvhYTNMErQsNSJZ0wr91ucOmL5JI'
-SIGNATURE = 'g1L73Mv6GnUE8vfWHmxKQmG9KuLiwwI9LMR50w+SXTo='
+CODE = b'rp6YlWEcSxS81OTmdksLOaqv7TRbpzSV'
+ACCESS_TOKEN_URL = b'https://devshop.epages.com/rs/shops/lingering-hill-2037/token'
+CLIENT_SECRET = b'pM8vwvhYTNMErQsNSJZ0wr91ucOmL5JI'
+SIGNATURE = b'g1L73Mv6GnUE8vfWHmxKQmG9KuLiwwI9LMR50w+SXTo='
```

Comment:
See comment above, please value the method contract and encode strings to bytestrings in the method